### ISSUE TITLE
OpenShift Support

### ISSUE THREAD

**Issue by Jotschi** [gentics/mesh#987]
To better support OpenShift change:

Change group permission:
* chgrp root /graphdb 

Verify that Gentics Mesh container runs with any uid.

Example:
```docker run -u 120552 gentics/mesh:1.3.2-guidfix```

**Comment by Jotschi** [comment]
```
+RUN chown mesh:root /graphdb    -R && chmod g+rw /graphdb  -R && \
+    chown mesh:root /uploads    -R && chmod g+rw /uploads  -R && \
+    chown mesh:root /backups    -R && chmod g+rw /backups  -R && \
+    chown mesh:root /plugins    -R && chmod g+rw /plugins  -R && \
+    chown mesh:root /keystore   -R && chmod g+rw /keystore -R && \
+    chown mesh:root /config     -R && chmod g+rw /config   -R && \
+    chown mesh:root /mesh       -R && chmod g+rw /mesh     -R 
 
```

**Comment by Jotschi** [comment]
Released with 1.3.3
